web/src/LocalStorage.tsx

```diff
@@ -34,20 +34,35 @@ export function accessorsForTesting<S>(name: string) {
 }
 
 // Like `useState`, but backed by localStorage and namespaced by the tiltfileKey
-export function usePersistentState<S>(name: string, defaultValue: S) {
+// prepareSavedState: transforms any state read from storage - allows, e.g., filling in default values for
```

Comment:
hmm, maybe `normalizeSavedState`? `backfillSavedState`? `validateSavedState`? i don't love any of these (including yours) but don't hate it, the comment is enough to make sure I know what it does

Reply:
I think I like `maybeUpgradeSavedState`